bin/control.sh

```diff
@@ -61,7 +61,10 @@ case ${COMPONENT} in
                 FLYWAY_EXECUTABLE="$FLYWAY_DIR/flyway"
                 SCRIPTS_LOCATION="$FLYWAY_DIR/$DATABASE_SCRIPT_DIR"
 
+                echo FLYWAY_LOCATIONS="filesystem:$SCRIPTS_LOCATION" "$FLYWAY_EXECUTABLE" migrate -url="$DB_URL" -user="$DB_USERNAME" -password="" || exit 1
```

Comment:
This does the job, but there's a risk of changing the command without changing the 'echo'd command.

How about storing the command in a variable? Then you can execute and echo it. When you echo it you will have to set unse the password with `env` `env PASSWORD=hidden echo command`

Reply:
If we store the command in variable, that will also be displayed because of `set -x`